cmd/podman/containers/rm.go

```diff
@@ -91,6 +94,12 @@ func init() {
 }
 
 func rm(cmd *cobra.Command, args []string) error {
+	if cmd.Flag("time").Changed {
+		if !rmOptions.Force {
+			return errors.New("you must specify --force if you are going to specify timeouterror reading CIDFile")
```

Comment:
```suggestion
			return errors.New("--force option must be specified to use the --time option")
```
I like that more (from the man page).